Content.Server/Commands/CommandUtils.cs

```diff
@@ -49,42 +48,5 @@ public static class CommandUtils
             attachedEntity = session.AttachedEntity.Value;
             return true;
         }
-
-        public static string SubstituteEntityDetails(IConsoleShell shell, EntityUid ent, string ruleString)
-        {
-            var entMan = IoCManager.Resolve<IEntityManager>();
-            var transform = entMan.GetComponent<TransformComponent>(ent);
-
-            // gross, is there a better way to do this?
-            ruleString = ruleString.Replace("$ID", ent.ToString());
-            ruleString = ruleString.Replace("$WX",
-                transform.WorldPosition.X.ToString(CultureInfo.InvariantCulture));
-            ruleString = ruleString.Replace("$WY",
-                transform.WorldPosition.Y.ToString(CultureInfo.InvariantCulture));
-            ruleString = ruleString.Replace("$LX",
-                transform.LocalPosition.X.ToString(CultureInfo.InvariantCulture));
-            ruleString = ruleString.Replace("$LY",
-                transform.LocalPosition.Y.ToString(CultureInfo.InvariantCulture));
-            ruleString = ruleString.Replace("$NAME", entMan.GetComponent<MetaDataComponent>(ent).EntityName);
-
-            if (shell.Player is IPlayerSession player)
-            {
-                if (player.AttachedEntity is {Valid: true} p)
-                {
-                    var pTransform = entMan.GetComponent<TransformComponent>(p);
-
-                    ruleString = ruleString.Replace("$PID", ent.ToString());
-                    ruleString = ruleString.Replace("$PWX",
-                        pTransform.WorldPosition.X.ToString(CultureInfo.InvariantCulture));
-                    ruleString = ruleString.Replace("$PWY",
-                        pTransform.WorldPosition.Y.ToString(CultureInfo.InvariantCulture));
-                    ruleString = ruleString.Replace("$PLX",
-                        pTransform.LocalPosition.X.ToString(CultureInfo.InvariantCulture));
-                    ruleString = ruleString.Replace("$PLY",
-                        pTransform.LocalPosition.Y.ToString(CultureInfo.InvariantCulture));
-                }
-            }
-            return ruleString;
-        }
```

Comment:
Is this intentional?

Reply:
Yes, it's unused. It made more sense to me to remove it than to refactor it.